data/categories.txt

```diff
@@ -6899,6 +6927,7 @@ fa:تلفن اضطراری
 
 leisure-fitness_centre
 en:Fitness Centre|Gym|U+1F4AA|fitness|workout|fitness gym|fitness club|health club|fitness studio|workout studio
+en-US:Fitness Center|Gym|U+1F4AA|fitness|workout|fitness gym|fitness club|health club|fitness studio|workout studio
```

Comment:
@vng the same question here: can we remove all search synonyms for the regional version? Will the search find it with en-US locale if a user types gym?

Reply:
In this case, gym is in both (albeit incorrectly capitalised).